tests/roles/barbican_adoption/defaults/main.yaml

```diff
@@ -17,8 +42,47 @@ barbican_patch: |
           database: BarbicanDatabasePassword
           service: BarbicanPassword
           simplecryptokek: BarbicanSimpleCryptoKEK
+        {% if barbican_hsm_enabled %}
```

Comment:
I'm confused - if we're using hsm_specific templates for the patches , why do we need this here?